File: MATLAB/ASCII_30_Del.m
function y = ASCII_30_Del(x)
% For use on the substituted German alphabet.  Revert the changes to
% retrieve the native German characters.
    n = size(x, 2);
    for i = 1 : n
        if (x(i) == 123) 
            y(i) = 228;
        elseif (x(i) == 124)
            y(i) = 246;
        elseif (x(i) == 125)
            y(i) = 252;
        elseif (x(i) == 126)
            y(i) = 223;
        else
            y(i) = x(i);
        end
    end
end

File: Julia/ASCII_30_Del.jl
function ASCII_30_Del(dectext_ascii_30)
    dectext_ascii_30[find(x -> x == 123, dectext_ascii_30)] = 228
    dectext_ascii_30[find(x -> x == 124, dectext_ascii_30)] = 246
    dectext_ascii_30[find(x -> x == 125, dectext_ascii_30)] = 252
    dectext_ascii_30[find(x -> x == 126, dectext_ascii_30)] = 223
    return dectext_ascii_30
end